openbas-front/src/admin/components/assets/security_platforms/SecurityPlatform.d.ts

```diff
@@ -1,6 +1,5 @@
 import type { SecurityPlatform } from '../../../../utils/api-types';
 
-export type SecurityPlatformStore = Omit<SecurityPlatform, 'asset_executor', 'asset_tags'> & {
+export type SecurityPlatformStore = Omit<SecurityPlatform, 'asset_tags'> & {
```

Comment:
**suggestion:** Can be removed because SecurityPlatform already have asset_tags?: string[]